akka-javasdk/src/main/scala/akka/javasdk/impl/workflow/ReflectiveWorkflowRouter.scala

```diff
@@ -80,16 +80,15 @@ class ReflectiveWorkflowRouter[S, W <: Workflow[S]](
         case payload if payload.nonEmpty => serializer.fromBytes(payload).asInstanceOf[S]
       }
 
-  // in same cases, the runtime may send a message with contentType set to object.
-  // if that's the case, we need to patch the message using the contentType from the expected input class
   private def decodeInput(result: BytesPayload, expectedInputClass: Class[_]) = {
```

Comment:
Since we don't support `result` anymore, maybe it's better to rename it to `input`?